app/src/main/java/com/infomaniak/mail/data/cache/MailboxContentController.kt

```diff
@@ -95,6 +95,9 @@ object MailboxContentController {
         MailRealm.mailboxContent.writeBlocking { getLatestMessage(uid)?.let(::delete) }
     }
 
+    fun getLatestMessage(uid: String): Message? =
+        MailRealm.mailboxContent.writeBlocking { getLatestMessage(uid) }
```

Comment:
Why `getLatestMessage` is in `writeBlocking ` ?

Reply:
There is 2 `getLatestMessage`:
- the public one
- the private one that is an extension function of `MutableRealm`, so it needs to be called in a `write` block.

Reply:
okay 👍🏾